src/plugins/links/public/lib/serialize_attributes.ts

```diff
@@ -14,9 +14,14 @@ export const serializeLinksAttributes = (
   state: LinksRuntimeState,
   shouldExtractReferences: boolean = true
 ) => {
-  const linksToSave: Link[] | undefined = state.links?.map(
-    ({ title, description, error, ...linkToSave }) => linkToSave
-  );
+  const linksToSave: Link[] | undefined = state.links
+    ?.map(({ title, description, error, ...linkToSave }) => linkToSave)
+    ?.map(
```

Comment:
Can we add a comment why we're doing this? Maybe something like this?

```suggestion
    // Remove any properties with null values that may have been serialized when backing up runtimeState
    ?.map(
```

Reply:
Good call, added in b687476b9e253064e7074e3d002f539ee01bb2a5